packages/admin/cms-admin/src/index.ts

```diff
@@ -90,6 +90,7 @@ export type { SiteConfig } from "./sitesConfig/SitesConfigContext";
 export { SitesConfigProvider } from "./sitesConfig/SitesConfigProvider";
 export { useSiteConfig } from "./sitesConfig/useSiteConfig";
 export { useSitesConfig } from "./sitesConfig/useSitesConfig";
+export { CurrentUserContext, CurrentUserInterface, CurrentUserProvider, useCurrentUser } from "./userPermissions/hooks/currentUser";
```

Comment:
Do we need to export the context if we export the provider and hook wrappers?

Reply:
76f3e2bba151574e003e9db0da4e0733f7974c90